pkg/resources/pciNetDevice.go

```diff
@@ -42,6 +43,7 @@ func NewPciNetDevice(pciDevice *ghw.PCIDevice, rFactory types.ResourceFactory) (
 	dSpecs := infoProvider.GetDeviceSpecs(pciAddr)
 	mnt := infoProvider.GetMounts(pciAddr)
 	env := infoProvider.GetEnvVal(pciAddr)
+	rdmaSpec := rFactory.GetRdmaSpec(pciDevice.Vendor.ID, pciDevice.Address)
```

Comment:
@Mmduh-483 as this is done correctly, you didn't use the driver parameter, hence driver should not be specified as I explained in previous comment.